rasa/shared/utils/pykwalify_extensions.py

```diff
@@ -20,7 +20,7 @@ def require_response_keys(
             # this is handled by other validation rules
             continue
 
-        if not response.get("text") and not response.get("custom"):
+        if not (response.get("text") is not None) and not response.get("custom"):
```

Comment:
mhm this is odd, what's the `not is not` for, so should it be

```
if response.get("text") is None and not response.get("custom"):

Reply:
Aha, good point! I was just processing the information in a weird way 🤔 
```suggestion
        if response.get("text") is None and not response.get("custom"):
```